corehq/apps/users/migrations/0031_add_email_reports_permission.py

```diff
@@ -0,0 +1,31 @@
+from django.db import migrations
+
+from dimagi.utils.couch.database import iter_docs
+
+from corehq.apps.users.models import UserRole
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def migrate_email_reports_permissions(apps, schema_editor):
+    roles = UserRole.view(
+        'users/roles_by_domain',
+        include_docs=False,
+        reduce=False
+    ).all()
+    for role_doc in iter_docs(UserRole.get_db(), [r['id'] for r in roles]):
```

Comment:
Are you planning to run this in a private release before deploy?

Reply:
I can shoot for that-- what are the benefits of it?

Reply:
Re-reading, does this need a migration at all? You're setting `email_reports` to true, but that's the default value anyway. This is analogous to this PR: https://github.com/dimagi/commcare-hq/pull/27495/

@snopoke or @esoergel does that sound right? I always like to get another opinion on migrations decisions.

Reply:
Permissions are now fully read from SQL so there is no way to default a new permission to True in code without doing a migration. This migration (the one in this PR) will work since the sync from couch to SQL is present but I think it would be better to convert this to a SQL migration:

```python
new_permission_name = 'email_reports'
permission = SQLPermission.objects.get_or_create(value=new_permission_name)
for role in SQLUserRole.objects.all().iterator():
    role.rolepermission_set.get_or_create(permission_fk=permission, defaults={"allow_all": True})
    role._migration_do_sync()
```

Notes:
- you could do it in bulk but that would not sync the data back to couch (which we do still want at this point)
- above code is untested
- maybe we can make a pattern out of this to make it easier in future to do permission migrations

Reply:
Thank you @snopoke-- I modified the original migration I had here slightly, then converted it in this commit: 6855dfa